tests/foreman/api/test_capsulecontent.py

```diff
@@ -816,7 +816,7 @@ def test_positive_sync_kickstart_repo(
             basearch='x86_64',
             org_id=function_entitlement_manifest_org.id,
             product=constants.REPOS['kickstart'][distro]['product'],
-            reposet=constants.REPOSET['kickstart'][distro],
+            reposet=constants.REPOS['kickstart'][distro]['reposet'],
```

Comment:
Instead of updating this, we can directly update parametrized marker 
from `@pytest.mark.parametrize('distro', ['rhel7', 'rhel8_bos', 'rhel9_bos'])` 
to
`@pytest.mark.parametrize('distro', ['rhel7', 'rhel8', 'rhel9'])`
what do you think?